### ISSUE TITLE
Display commands to run after adding the package

### ISSUE THREAD

**Issue by Artusamak** [Happyculture/combawa#1]
When you require the package, you don't known what to do (unless you read the README), i kind addition whould be to add a message to list the commands that can be run.

**Comment by Artusamak** [comment]
Composer doesn't let us run commands from the packages due to security reasons so we can't add a custom script that would kindly suggest to use Combawa commands.

We must let this in the doc.

**Comment by DuaelFr** [comment]
Composer `post-install-cmd` script should be run as it is a "root package"

See https://getcomposer.org/doc/articles/scripts.md#what-is-a-script-

**Comment by Artusamak** [comment]
Yes so it means that people would have to declare the script in their composer.json. So it's a dead end. If they have to tweak the composer.json file to be warned how to use Combawa, they know about Combawa usage ^^ (because they will have read the README). That's why i'm saying that we should keep the doc clear (and that we can't do more due to Composer limitation).

For the record I have a README update incoming.

**Comment by DuaelFr** [comment]
Combawa **is** a root package. A root package is a package that has directly been `composer require`'d in the project in opposition to a dependency of a root package.

**Comment by DuaelFr** [comment]
I spent a few hours on this today.
It does not seem doable for now even if the documentation says it is.
It would need our project to be of type `composer-plugin` instead of a `drupal-console-library` but it wouldn't be detected by Drupal console anymore...

**Comment by DuaelFr** [comment]
This will be possible when we'll drop Drupal Console for Drush because drush does not need our package to be of a specific type.

**Comment by DuaelFr** [comment]
Blocked by #70

**Comment by Artusamak** [comment]
To reevaluate.
